src/box/lua/config/utils/aboard.lua

```diff
@@ -0,0 +1,149 @@
+-- Alert board.
+
+local datetime = require('datetime')
+local log = require('internal.config.utils.log')
+
+-- Set an alert.
+--
+-- The `key` is an unique identifier of the alert. Attempt to set
+-- a new alert with the same key replaces the existing alert.
+--
+-- The key can be used to drop the alert using :drop().
+--
+-- The `alert` is a table of the following structure.
+--
+-- {
+--     type = 'warn' or 'error',
+--     message = <string>,
+-- }
+--
+-- More fields may be placed into the table and all these fields
+-- are returned from :alerts().
+--
+-- The alert board takes an ownership of the `alert` table and
+-- modifies it.
+--
+-- A timestamp is saved on the :set() method call to show it in
+-- an :alerts() result.
+local function aboard_set(self, key, alert)
+    assert(alert.type == 'error' or alert.type == 'warn')
+    if alert.type == 'error' then
+        log.error(alert.message)
+    else
+        log.warn(alert.message)
+    end
+    alert.timestamp = datetime.now()
+    self._alerts[key] = alert
+end
+
+-- Return an alert pointed by the given `key` or nil.
+local function aboard_get(self, key)
+    return self._alerts[key]
+end
```

Comment:
I would keep it for stylistic purposes: it is easier to switch from one description to another, when all of them are present. Otherwise, I need to switch my brain into the Lua interpreter mode :)